de_templates/ingest/Ingest_AzureSql_SourceName/data_factory/pipelines/Ingest_AzureSql_SourceName.json.jinja

```diff
@@ -0,0 +1,237 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "factoryName": {
+            "type": "string",
+            "metadata": "Data Factory name"
+        }
+    },
+    "variables": {
+        "factoryId": "[concat('Microsoft.DataFactory/factories/', parameters('factoryName'))]"
+    },
+    "resources": [
+        {
+            "name": "[concat(parameters('factoryName'), '/Ingest_AzureSql_{{ source_name }}')]",
+            "type": "Microsoft.DataFactory/factories/pipelines",
+            "apiVersion": "2018-06-01",
+            "properties": {
+                "description": "Ingest from demo Azure SQL database using ingest config file.",
```

Comment:
Description needs to be dynamic / taken from a parameter.

Reply:
done